src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -518,14 +514,6 @@ public void ExecuteCommand(ReturnKeyCommandArgs args, Action nextHandler)
                 return;
             }
 
-            // Finally, wait and see if completion is computing. If it is, we want to allow
-            // the list to pop up rather than insert a blank line in the buffer.
-            if (_completionService.WaitForComputation(args.TextView, args.SubjectBuffer))
```

Comment:
This was very tricky at the time and the scenario is quite old (the code pre-dates the move to git). I think the overall change makes sense. We'll just need to watch out for this issue and see if it comes up again.